src/Libraries/RevitNodes/Elements/CurveElement.cs

```diff
@@ -114,7 +114,12 @@ protected static void setCurveMethod(Autodesk.Revit.DB.CurveElement ce, Curve c)
         }
 
         #endregion
-
+        /// <summary>
+        /// Transforms a smooth curve into an approximate polygonal representation with a given tolerance for max length
+        /// </summary>
+        /// <param name="package"></param>
+        /// <param name="tol"></param>
+        /// <param name="gridLines"></param>
         public new void Tessellate(IRenderPackage package, double tol, int gridLines)
```

Comment:
Do we show this node in library? It should not be shown. Can you please hide it from library view.